roles/static_inventory/tasks/checkpoint.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: create static inventory dir
+  file:
+    path: "{{ inventory_path }}"
```

Comment:
Can we replace this with `"{{ inventory_dir }}"` instead? That way, we'll just add `hosts` to the existing inventory the user already set up. I think that will be less confusing than creating a new directory somewhere else.